client/src/components/Form.js

```diff
@@ -32,12 +32,12 @@ const ButtonIcon = styled.button`
   }
 `;
 
-export const FormInput = ({ title, icon, alt }) => {
+export const FormInput = ({ title, icon, alt, onClick, onChange }) => {
```

Comment:
Taking a look at the usage of this component, I would suggest another change:

```suggestion
export const FormInput = ({ title, icon, alt, onSubmit, onChange }) => {
```

Remove `onClick` and set `<Form onSubmit={handleSubmit}>`

```js
const handleSubmit = (event) => {
  event.preventDefault();
  onSubmit();
}
```

Reply:
Not related to this pull request, but `value` is missing.
I would suggest removing `value` and `onChange` completely and add a state in this component.

This will reduce the complexity 😎